app.js

```diff
@@ -26,16 +26,34 @@ app.get('/api/v1/projects', async (req, res) => {
     }, []);
     res.status(200).json(projsWithPalettes);
   } catch (error) {
-    res.status(404).json({ error: 'No projects found' });
+    res.status(500).json({ error });
   };
 });
 
 app.get('/api/v1/palettes', async (req, res) => {
+  const { color } = req.query;
+  if (color) {
+    try {
+      let queriedColor = await database('palettes').where('color1', `#${color}`)
+        .orWhere('color2', `#${color}`)
```

Comment:
Like this `.orWhere`, have not seen that before. I assume that is a why of iterating over until a condition matches?